shoop/admin/modules/orders/views/create.py

```diff
@@ -86,28 +144,84 @@ def handle_product_data(self, request):
         product_id = request.GET["id"]
         shop_id = request.GET["shop_id"]
         customer_id = request.GET.get("customer_id")
+        quantity = decimal.Decimal(request.GET.get("quantity", 1))
         product = Product.objects.get(pk=product_id)
+        if not product:
```

Comment:
This code will never be reached; if `.get` errors out, it'll raise an ODNE, not return None.